seaborn/_core/properties.py

```diff
@@ -619,7 +620,14 @@ def infer_scale(self, arg: Any, data: Series) -> Scale:
         if arg in QUAL_PALETTES:
             return Nominal(arg)
         elif var_type == "numeric":
-            return Continuous(arg)
+            # Prioritize actual colormaps, e.g. if a colormap named "pow" exists
+            if is_registered_colormap(arg):
+                return Continuous(arg)
+            elif any(arg.startswith(k) for k in self._TRANS_ARGS):
```

Comment:
Should this be `startswith` or `==` ? With the current code, passing `"logfoo"` for instance would redirect it to `trans`; is that what we want ?

Reply:
Thanks for the comment. I didn't want to change this since it is similar to how it's performed elsewhere in this module and in other parts of the codebase (`_core.scales` for example). I reckon the original intention in using `startswith` was to support "log2", "symlog100", etc. Anyway, passing "logfoo" would fail even prior to this PR (with `ValueError: 'logfoo' is not a valid palette name`).

Reply:
+1, the prefix checks allow the "string parameterized scale transform" spelling but maybe that needs better error checking.